tasks/main.yml

```diff
@@ -20,7 +20,7 @@
   tags:
     - haproxy
 
-- name: Start Marathon service
-  service: name=marathon state=restarted enabled=yes
+- name: Enable Marathon service
+  service: name=marathon enabled=yes
```

Comment:
OK will do!
(though I'm sure marathon is already restarted on 1. install (configs are created) OR on any other config change )